check-module-version/README.md

```diff
@@ -0,0 +1,42 @@
+# GitHub Action: Check module version
+
+This action checks the equality of the version from Lua 
+module's code and the last repository tag.
```

Comment:
Now we check tag that had triggered the pipeline, not the last one

Reply:
Updated